tardis/tests/tests_slow/plot_helpers.py

```diff
@@ -0,0 +1,32 @@
+import tempfile
+from pytest_html import extras
+import tardis
+
+
+class PlotUploader(object):
```

Comment:
I'd add an `add` method for adding plots and call `upload()` in `pytest_runtest_makereport`. Then we can give the report object to `upload()` and `upload` adds stuff like the `passed/failed` text depending on the report